zhaquirks/tuya/ts0601_motion.py

```diff
@@ -327,6 +327,65 @@ class TuyaHumanMotionState(t.enum8):
     .add_to_registry()
 )
 
```

Comment:
We have standardized on two spaces between quirks

Reply:
fixed